packages/pagebuilder/lib/ContentTypes/Banner/banner.js

```diff
@@ -274,8 +274,8 @@ const Banner = props => {
 
     const history = useHistory();
 
-    const clickHandler = e => {
-        handleHtmlContentClick(history, e);
+    const clickHandler = (event, isClick = false) => {
```

Comment:
Instead of relying on a second parameter, can't we validate click using `event.type`? https://developer.mozilla.org/en-US/docs/Web/API/Event/type

Reply:
Good catch, fixing this now!

Reply:
@pedrochiossi ✅